expression/builtin_time.go

```diff
@@ -5979,9 +5979,13 @@ func (b *builtinSecToTimeSig) evalDuration(row chunk.Row) (types.Duration, bool,
 		second = seconds % 60
 	}
 	secondDemical = float64(second) + demical
+	secStr, err := types.ConvertScientificNotation(fmt.Sprintf("%v", secondDemical))
```

Comment:
Could `strconv.FormatFloat` be used here?
https://play.golang.org/p/JDZU_za0JoV

Reply:
A nice suggestion! I have updated this PR to use this method and reverted `convert.go`.